### PR TITLE
Add alumni role

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | See https://github.com/PrestaShop/open-source/issues/92
| Fixed ticket | Fixes https://github.com/PrestaShop/open-source/issues/92

### CODE REVIEW

**Review by Hlavtox — APPROVED** [review]

**Comment by matks** [comment]
Thanks @Hlavtox 😉 I'll wait a little before merging to gather a little more feedback

**Review by matthieu-rolland — APPROVED** [review]
nice concept 👍

**Review by kpodemski — APPROVED** [review]

**Comment by matks** [comment]
I got not negative feedbacks from anybody I shared it with, I have been talking about it with people for a week now, and it has 3 approvals: I think we can say it's a YES by lazy consensus

**Pull request merged**
